common/src/test/java/org/apache/drill/test/DrillTest.java

```diff
@@ -54,7 +54,7 @@
   static MemWatcher memWatcher;
   static String className;
 
-  @Rule public final TestRule TIMEOUT = TestTools.getTimeoutRule(100000);
+  @Rule public final TestRule TIMEOUT = TestTools.getTimeoutRule(180000);
```

Comment:
Two comments. One a nit: I find it hard to read 180000. Suggestion, use 180_000.

More substantial comment: 180K ms = 180 seconds = 3 minutes. This seems FAR too long for a default timeout.

I wonder, should we make this much smaller? Then, find the tests that need extra time and mark them specially? Maybe tag them with "SlowTests" and set a larger limit for just those tests?

This will cause tests to fail if they are too slow. That, in turn, will force us to either fix those tests, or label them as slow.